### PR TITLE
feat: 🎸 Add identities namespace

### PR DESCRIPTION
**Pull request by polymath-eric**
Creates an identities namespace and moves methods into it

- sdk.registerIdentity -> sdk.identities.registerIdentity
- sdk.getIdentity -> sdk.identities.getIdentitiy
- sdk.isIdentityValid -> sdk.identities.isIdentityValid

BREAKING CHANGE: 🧨 Moves Identity methods

### CODE REVIEW

**Review by prashantasdeveloper — COMMENTED** [review]
LGTM

**Comment by polymath-eric** [comment]
@monitz87 Prashant gave an approval already for this one.

**Review by prashantasdeveloper — APPROVED** [review]

File: src/Identities.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { Context, Identity, registerIdentity, RegisterIdentityParams } from '~/internal';
+import { ProcedureMethod } from '~/types';
+import { createProcedureMethod } from '~/utils/internal';
+
+/**
+ * Handles all Identity related functionality
+ */
+export class Identities {
```
Comment (monitz87) [inline src/Identities.ts:8]: Shouldn't `getCurrentIdentity` be in this namespace as well?

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/Identities.ts (hunk 1 above)
Comment (polymath-eric) [reply src/Identities.ts:8]: To me the idea of "currentIdentity" has to do more with SDK configuration than with interacting with Identities. 

In a sense the corresponding "setter" is the `setSigner` method and I think they should be kept together. The usage of the "currentIdentity" is different than all of the other methods here imo.

That said I seem to be in the minority here so I can move it if you still think it belong in the namespace.

Comment (monitz87) [reply src/Identities.ts:8]: You make a very good point. Let's leave it as is for now

**Review by monitz87 — APPROVED** [review]

**Pull request merged**